scripts/stage-release.sh

```diff
@@ -173,9 +178,11 @@ git checkout -b "${PREPARE_DEVELOPMENT_BRANCH}" "${TEMPORARY_RELEASE_BRANCH}"
 
 updateVersions "${RELEASE_VERSION}" "${DEVELOPMENT_VERSION}"
 # bump the Changelog to the next SNAPSHOT version. We do it this way so the changelog has the new release as the first entry
-${SED} -i -e "s_##\s${RELEASE_VERSION//./\\.}_## SNAPSHOT\n## ${RELEASE_VERSION//./\\.}_g" CHANGELOG.md
+replaceInFile "s_##\s${RELEASE_VERSION//./\\.}_## SNAPSHOT\n## ${RELEASE_VERSION//./\\.}_g" CHANGELOG.md
 
-git add 'CHANGELOG.md'
+# bump the docs for the development version
+replaceInFile "s_:ProductVersion:.*_:ProductVersion: ${DEVELOPMENT_VERSION%.*}_g" docs/_assets/attributes.adoc
+replaceInFile "s_:tag:.*_:tag: v${DEVELOPMENT_VERSION//./\\.}_g" docs/_assets/attributes.adoc # this doesn't make a lot sense...
```

Comment:
Not entirely sure this is the right move as the tag will never exist but then again there is no valid answer that will work.

Reply:
maybe `main` is better at development time? then the dev docs would have a working link for:

```
27::design-doc: https://github.com/kroxylicious/kroxylicious/blob/{tag}/kroxylicious-filters/kroxylicious-encryption/doc/design.adoc
28::start-script: https://github.com/kroxylicious/kroxylicious/blob/{tag}/kroxylicious-app/src/assembly/kroxylicious-start.sh
```

hmm, `kroxylicious-encryption` should also be `kroxylicious-record-encryption` in that design-doc link and I can't find a usage of `{design-doc}`, maybe that attribute is dead?

also, the link to releases, does it have to point at a valid tag?

```
modules/con-operating.adoc
15:The Kroxylicious {github-releases}[binary distributions^] ship with https://logging.apache.org/log4j/2.x[log4j2] as a logging backend. To customize the logging configuration:
```
maybe we could link to `https://github.com/kroxylicious/kroxylicious/releases` without the tag for that particular link.

Reply:
If we do that it might be better renamed something like `gitRef`

Reply:
I think I'd just remove the hyperlink from that sentence.  It doesn't really add anything of value.